src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -231,4 +231,31 @@ public void testCreateAllArgsTeam() throws IOException {
         assertThat(team.getDescription(), equalTo("Team description"));
         assertThat(team.getPrivacy(), equalTo(GHTeam.Privacy.CLOSED));
     }
+
+    @Test
+    public void testOrganizationHasProjectsEnabled() throws IOException {
+        // Initially projects are enabled
+        // Arrange
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+
+        // Act
+        boolean result = org.hasProjectsEnabled();
+
+        // Assert that projects are enabled
+        assertThat(result, is(true));
+
+        // Disable projects
+        // Arrange
+        org.root.createRequest()
+                .withUrlPath("/orgs/" + GITHUB_API_TEST_ORG)
+                .method("PATCH")
+                .with("has_organization_projects", false)
+                .fetchInto(org);
```

Comment:
Since this exists you might as well add a public method to `GHOrganization` to update this value and test that as well.